src/search/landmarks/landmark_count_heuristic.cc

```diff
@@ -30,6 +30,8 @@ namespace landmarks {
 LandmarkCountHeuristic::LandmarkCountHeuristic(const plugins::Options &opts)
     : Heuristic(opts),
       use_preferred_operators(opts.get<bool>("pref")),
+      use_preferred_operators_conjunctive(opts.get<bool>("use_preferred_operators_conjunctive")),
```

Comment:
I assume this option (and the one just below) is going to be removed before merging.